gui/js/onboarding.window.js

```diff
@@ -36,6 +36,24 @@ module.exports = class OnboardingWM extends WindowManager {
     }
   }
 
+  jumpToSyncPath () {
+    this.shouldJumpToSyncPath = true
+    this.send('registration-done')
+    this.win.once('dom-ready', () => {
+      setTimeout(() => this.send('registration-done')
+      , 20)
```

Comment:
Add `TODO: Ensure elm side sends something through ports so we can trigger 'registration-done' without relying on timeouts`?

Reply:
(although timeout was already present in code moved from `gui/main`)